### DISCUSSION TITLE
Check if player crashed

### DISCUSSION THREAD

**Discussion by NjuuOwO** [YimMenu/YimMenu#2137] [Q&A]
Is there any way to check if a player is crashed (or force quited the game)? Maybe these stop getting send?
![image](https://github.com/YimMenu/YimMenu/assets/45913099/7ca32876-b5fb-4eb1-b4ef-f3d40d2a61f8)

**Comment by maybegreat48** [comment]
Well yes we can see if they timed out of the game
